content/scout/policy/_index.md

```diff
@@ -152,7 +152,7 @@ provenance attestation with max mode. To ensure compliance,
 update your build command to attach these attestations at build-time:
 
 ```console
-$ docker buildx build --provenance=true --sbom=true -t <IMAGE> --push .
+$ docker buildx build --provenance=mode=max --sbom=true -t <IMAGE> --push .
```

Comment:
Max is the default mode

Reply:
Not sure how that got changed, I'm reverting it, thank you.